library/src/main/res/values/attrs.xml

```diff
@@ -10,7 +10,9 @@
         <attr name="zoomEnabled" format="boolean" />
         <attr name="flingEnabled" format="boolean" />
         <attr name="scrollEnabled" format="boolean" />
-        <attr name="singleFingerScrollEnabled" format="boolean" />
+        <attr name="oneFingerScrollEnabled" format="boolean" />
+        <attr name="twoFingersScrollEnabled" format="boolean" />
```

Comment:
do you really want to go with finger vs finger**s**?
`twoFingerScrollEnabled` would be ok in my brain and it would be more consistent only changing the count.

Reply:
While it would make sense to go with `twoFingerScrollEnabled`, I think we should still go with `twoFingersScrollEnabled` since @natario1 specifically suggested here https://github.com/natario1/ZoomLayout/pull/102#issuecomment-478974434